### PR TITLE
[Routing] [FrameworkBundle] Added --sort option to debug:router

### PR DESCRIPTION
**Pull request by vjandrea**
Documentation for this PR: symfony/symfony#34565

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -350,6 +350,23 @@ evaluates them:
     blog_show         ANY      ANY      ANY    /blog/{slug}
     ----------------  -------  -------  -----  --------------------------------------------
 
+You can sort the routes by name or path using the --sort option
```
Comment (OskarStark) [inline routing.rst]: —sort in double backticks please

Comment (vjandrea) [reply routing.rst]: Done, thanks @OskarStark

**Comment by OskarStark** [comment]
Please reopen when you reopen the PR

**Comment by javiereguiluz** [comment]
I'm closing this because the related code PR was closed.

**Pull request closed without merging**